.all-contributorsrc

```diff
@@ -344,6 +344,15 @@
         "code",
         "example"
       ]
+    },
+    {
```

Comment:
Please just stick to the instructions given in the issue, once your PR get merged you will be added to the contributors.

Reply:
Noted!